packages/shared/src/components/auth/common.tsx

```diff
@@ -65,6 +67,33 @@ export const AuthModalText = classed(
   'typo-body text-theme-label-secondary',
 );
 
+interface DiscardAuthProps {
+  onContinue: CloseModalFunc;
+  onExit: CloseModalFunc;
+  container?: HTMLElement;
+  isOpen: boolean;
+}
+
+export const DiscardAuthModal = ({
```

Comment:
Should this not get it's own file rather? (non blocking/ non important)

Reply:
I think this is an old code that could have already been removed by the modal update task. Let me check with Eliz how to use it.

Reply:
After checking with Eliz, she mentioned keeping the old implementation for the moment while things are still at work regarding modals. I think we can keep this one as it is since we will be removing this in the future. WDYT?